keyboards/peej/tripel/tripel.h

```diff
@@ -0,0 +1,82 @@
+/* Copyright 2021 Paul James
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define LEFT_PCB 1
+#define MIDDLE_PCB 2
+#define RIGHT_PCB 3
+
```

Comment:
```suggestion
#ifndef define PRO_MICRO_POSITION
#    define PRO_MICRO_POSITION LEFT_PCB
#endif

```

This way, it lets users select the correct one, via the keymap's config.h, but doesn't require them to undefine it first. 

Alternatively, you could use `post_config.h` to do this, as well.

Reply:
Makes sense. I hadn't thought about allowing the constant to be set from within the keymap's config.h, this makes it much better to configure the board.